package.json

```diff
@@ -47,6 +47,8 @@
 		"chokidar-cli": "=1.2.0",
 		"coveralls": "=2.13.1",
 		"cypress": "=0.20.3",
+		"chai": "=4.1.2",
+		"chai-as-promised": "=7.1.1",
```

Comment:
Let's create a follow-up issue to make more use of this.

Reply:
Created #560